src/test/java/hudson/plugins/buildblocker/BuildBlockerQueueTaskDispatcherTest.java

```diff
@@ -150,6 +142,15 @@ public void testMultipleExecutors() throws Exception {
         theJob1.delete();
     }
 
+    private void updateJobProperty(FreeStyleProject job, BuildBlockerProperty property) {
+        try {
+            job.removeProperty(BuildBlockerProperty.class);
+            job.addProperty(property);
+        } catch (IOException e) {
+
```

Comment:
should we add a log message or something? Just to be sure that we do not loose any important information.

Reply:
fixed